spec.md

```diff
@@ -1071,6 +1277,14 @@ message GetCapacityRequest {
   // specific `parameters`. These are the same `parameters` the CO will
   // use in `CreateVolumeRequest`. This field is OPTIONAL.
   map<string, string> parameters = 2;
+  
+  // If specified, the Plugin SHALL report the capacity of the storage
+  // that can be used to provision volumes that satisfy ALL of the
+  // specified `accessibility_requirements`. These are the same
+  // `accessibility_requirements` the CO will use in a
+  // `CreateVolumeRequest`.
+  // This field is OPTIONAL.
```

Comment:
and ... "This field SHALL NOT be set unless the plugin advertises the ACCESSIBILITY_CONSTRAINTS capability."